azurerm/data_source_data_lake_store.go

```diff
@@ -0,0 +1,68 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmDataLakeStoreAccount() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmDateLakeStoreAccountRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"resource_group_name": resourceGroupNameForDataSourceSchema(),
+
+			"location": locationForDataSourceSchema(),
+
+			"tier": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
I believe this should be `computed` with no default?

Reply:
yup has to be computed